<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>1500</width>
      <height>900</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../bg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>381</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../logo apppp123.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="us">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-50</y>
      <width>1321</width>
      <height>811</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 210, 191);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/310042565_1253044795483244_7626101413920627149_n.png</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>411</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{

border-radius:50px;
/*background-color: rgb(211, 174, 156);*/
background-color: rgb(255, 255, 255);


}

QLabel{
color: rgb(0,0,0);;
font-size:15px;

font-family:century gothic;
}

QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
font-size:rgb(0,0,0);
font-size:15px;
}

QPushButton{
font-size:22px;
font-family:century gothic;
background:rgb(255, 228, 228);
border-radius:60px;
color:#BF7069;
border-radius:15px;
/*border:1px solid #BF7069;*/
}
QPushButton:hover{
color:#931A43;
/*background:#801414;*/

background:transparent;
border-radius:15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="nom_sign">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Nom d'utilisateur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mail_sign">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>nom@esprit.tn</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mdp_sign">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhSensitiveData</set>
     </property>
     <property name="placeholderText">
      <string>******</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_sign">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
background-color:rgb(210, 175, 148);
	
border: none;

	font: italic 10pt &quot;Elephant&quot;;
text-align: left;
border: 1px solid #6a1616;
}
QComboBox:hover {
	color: rgb(255, 255, 255);

background:#6a1616;
border: none;
text-align: left;
}

</string>
     </property>
     <item>
      <property name="text">
       <string>salle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fournisseur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>evenement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>employe</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>420</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButtonr{
color:#931A43;
/*background:#801414;*/
font-family:century gothic;

background:rgb(255, 228, 228);

}
QPushButtonr{
font-size:22px;

background:rgb(255, 228, 228);
border-radius:15px;
border-radius:60px;
color:#BF7069;
border-radius:15px;
/*border:1px solid #BF7069;*/
}

QPushButtonr:hover{
color:#931A43;
/*background:#801414;*/
background:rgb(255, 228, 228);
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>sign up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Nom d'utilisateur:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;E-mail:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Mot de passe:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;gestion &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:22px;
font-family:century gothic;

border-radius:60px;
color:#BF7069;
border-radius:15px;
font-family:century gothic;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;sign UP&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;  Now!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="userr">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>401</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{

border-radius:50px;
/*background-color: rgb(211, 174, 156);*/
background-color: rgb(255, 255, 255);


}

QLabel{
color: rgb(0,0,0);;
font-size:15px;

font-family:century gothic;
}

QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
font-size:rgb(0,0,0);
font-size:15px;
}

QPushButton{
font-size:22px;
font-family:century gothic;
background:rgb(255, 228, 228);
border-radius:60px;
color:#BF7069;
border-radius:15px;
/*border:1px solid #BF7069;*/
}
QPushButton:hover{
color:#931A43;
/*background:#801414;*/

background:transparent;
border-radius:15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>***</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Nom d'utilisateur:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Mot de passe:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>400</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login In</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:22px;
font-family:century gothic;

border-radius:60px;
color:#BF7069;
border-radius:15px;
font-family:century gothic;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Now!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="user">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>620</y>
      <width>1321</width>
      <height>811</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>us</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAuthentification">
    <property name="title">
     <string>Authentification</string>
    </property>
   </widget>
   <addaction name="menuAuthentification"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
